### PR TITLE
Feature/#21 range component

### PR DESCRIPTION
**Pull request by Shinji22**

### CODE REVIEW

**Comment by vczb** [comment]
> _ No description provided. _

hi @Shinji22 I have already tested and your component is amazing! thanks for the contribution

before merging can you please rebase your git history? 

image

**Review by vczb — APPROVED** [review]
LGTM. waiting for rebasing

**Comment by Shinji22** [comment]
Hi @vczb,
Glad you like it :)
Sorry I messed up with my branch...
It should be OK now

File: src/packages/Range/stories.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react/types-6-0'
+import Range, { RangeProps } from '.'
+/*
```
Comment (vczb) [inline src/packages/Range/stories.tsx:4]: Hi! looks like you forgot this comment, please remove it. 

Also, revert the changes on the yarn.lock file

---

use rebase or amed to avoid create a new commit after the changes

Comment (vczb) [reply src/packages/Range/stories.tsx:4]: @Shinji22
